src/devices/feibit.ts

```diff
@@ -143,6 +143,112 @@ const definitions: Definition[] = [
+        model: 'TSKT114W-S1',
+        vendor: 'Feibit',
+        description: 'Four position situational switch',
+        extend: [deviceEndpoints({endpoints: {'1': 1, '2': 2, '3': 3, '4': 4}}), commandsOnOff({endpointNames: ['1', '2', '3', '4']})],
+    },
+    {
+        zigbeeModel: ['FNB56-ZRC06FB2.0'],
+        model: 'NZRC106W-M2',
+        vendor: 'Feibit',
+        description: 'Security controller',
+        extend: [],
+        fromZigbee: [fz.command_arm, fz.battery],
+        toZigbee: [],
+        exposes: [e.battery(), e.action(['panic', 'disarm', 'arm_day_zones', 'arm_all_zones'])],
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genBasic']);
+        },
+        onEvent: async (_, data) => {
+            if (data.type === 'commandArm' && data.cluster === 'ssIasAce') {
+                await data.endpoint.defaultResponse(0, 0, 1281, data.meta.zclTransactionSequenceNumber);
+            }
+        },
+    },
+    {
+        zigbeeModel: ['FEB56-ZT21AAK1.6'],
+        model: 'SZT211_AW-P1',
+        vendor: 'Feibit',
+        description: 'Curtain motor',
+        extend: [windowCovering({controls: ['lift', 'tilt']})],
+    },
+    {
+        zigbeeModel: ['FB56+TMT01ZY1.6'],
+        model: 'SSS401ZB-T',
+        vendor: 'Feibit',
+        description: 'Temperature control panel',
+        fromZigbee: [fz.thermostat, fz.on_off, fz.fan],
+        toZigbee: [
+            tz.thermostat_local_temperature,
+            tz.thermostat_occupied_heating_setpoint,
+            tz.thermostat_occupied_cooling_setpoint,
+            tz.thermostat_system_mode,
+            tz.on_off,
+            tz.fan_mode,
+        ],
+        exposes: [
+            e
+                .climate()
+                .withLocalTemperature()
+                .withSystemMode(['off', 'auto', 'cool', 'heat', 'fan_only', 'dry'])
+                .withFanMode(['off', 'low', 'medium', 'high', 'on', 'auto'])
+                .withSetpoint('occupied_heating_setpoint', 5, 30, 0.5)
+                .withSetpoint('occupied_cooling_setpoint', 5, 30, 0.5),
+            e.switch(),
+        ],
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['hvacThermostat', 'hvacFanCtrl', 'genOnOff']);
+            await reporting.thermostatOccupiedHeatingSetpoint(endpoint);
+            await reporting.thermostatTemperature(endpoint);
+            await reporting.fanMode(endpoint);
+            await reporting.onOff(endpoint);
+        },
+    },
+    {
+        zigbeeModel: ['FB56+ZSC02KJ1.0'],
+        model: 'TZSC302W-V1',
+        vendor: 'Feibit',
+        description: 'Dimmer Switch',
+        extend: [deviceEndpoints({endpoints: {'11': 11, '12': 12, '13': 13}}), light({powerOnBehavior: false})],
```

Comment:
```suggestion
        extend: [light({powerOnBehavior: false})],
```

I think this is a single device switch, does this work?

Reply:
Work very well, already optimized